[Issue] Using a config file with -nW fails

**Describe the bug**
When using the flags -config and -nW together it fails

**Subfinder version**
2.4.4

```
subfinder -config ./subfinder.config.yaml -d google.com -nW
```

I'm quite confident it doesnt have anything to do with my config file, it is working well without the -nW flag.
Also, having a very simple file seems to fail immediatly.

Output
```
        _     __ _         _
____  _| |__ / _(_)_ _  __| |___ _ _
(_-< || | '_ \  _| | ' \/ _  / -_) '_|
/__/\_,_|_.__/_| |_|_||_\__,_\___|_| v2.4.4

                projectdiscovery.io

[WRN] Use with caution. You are responsible for your actions
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[WRN] By using subfinder, you also agree to the terms of the APIs used.

[INF] Enumerating subdomains for google.com
panic: invalid argument to Intn

goroutine 1 [running]:
math/rand.(*Rand).Intn(0xc0000e69f0, 0x0, 0x1f)
        /usr/local/go/src/math/rand/rand.go:169 +0x9c
github.com/projectdiscovery/subfinder/pkg/resolve.(*ResolutionPool).getARecords(0xc0000e6a50, 0xc000026f20, 0x1f, 0x0, 0x0, 0x7ffe7f684639, 0xa, 0xc000026f20, 0x1f, 0xc000000180)
        /opt/go/src/github.com/projectdiscovery/subfinder/pkg/resolve/resolve.go:136 +0x15d
github.com/projectdiscovery/subfinder/pkg/resolve.(*ResolutionPool).InitWildcards(0xc0000e6a50, 0x7ffe7f684639, 0xa, 0xc0000e6a50, 0x0)
        /opt/go/src/github.com/projectdiscovery/subfinder/pkg/resolve/resolve.go:79 +0x165
github.com/projectdiscovery/subfinder/pkg/runner.(*Runner).EnumerateSingleDomain(0xc00000ed20, 0xa2bea0, 0xc0000240e8, 0x7ffe7f684639, 0xa, 0x0, 0x0, 0xc0000cbf00, 0x0, 0x0)
        /opt/go/src/github.com/projectdiscovery/subfinder/pkg/runner/enumerate.go:32 +0x1175
github.com/projectdiscovery/subfinder/pkg/runner.(*Runner).RunEnumeration(0xc00000ed20, 0xa2bea0, 0xc0000240e8, 0x0, 0x0)
        /opt/go/src/github.com/projectdiscovery/subfinder/pkg/runner/runner.go:44 +0x1e5
main.main()
        /opt/go/src/github.com/projectdiscovery/subfinder/cmd/subfinder/main.go:19 +0xb7

```

--- a/pkg/runner/initialize.go
+++ b/pkg/runner/initialize.go
@@ -55,9 +55,13 @@ func (r *Runner) initializeActiveEngine() error {
 
 	if r.options.Resolvers != "" {
 		resolvers = append(resolvers, strings.Split(r.options.Resolvers, ",")...)
-	} else {
+	} else if len(r.options.YAMLConfig.Resolvers) > 0 {
 		resolvers = append(resolvers, r.options.YAMLConfig.Resolvers...)
+	} else {
+		resolvers = append(resolvers, resolve.DefaultResolvers...)
 	}
+
 	r.resolverClient.AppendResolversFromSlice(resolvers)
+
 	return nil
 }

